hack/tools/release/notes/test/golden/v1.5.0.md

```diff
@@ -52,14 +52,14 @@ REPLACE ME: A couple sentences describing the deprecation, including links to do
 ## :bug: Bug Fixes
 - API: All: only set finalizers if deletionTimestamp is not set (#8949)
 - API: Ensure ownerReference apiVersions are always up to date (#8256)
-- CAPBK: Ignition: start kubeadm after network.target (#8772)
-- CAPBK: Set uninitialized taint only on worker nodes (#8358)
+- CABPK: Ignition: start kubeadm after network.target (#8772)
+- CABPK: Set uninitialized taint only on worker nodes (#8358)
```

Comment:
This is bc the the bug fix from https://github.com/kubernetes-sigs/cluster-api/pull/9814